### PR TITLE
ft(updating app) updating app js[#168404792]

### PR DESCRIPTION
**Pull request by irakozeyves1**
#### What does this PR do?
modification of app js with some element

#### Description of Task to be completed
this modification we adding the welcoming 
page , and the exit page when the routing is not 
found 

#### How should this be manually tested?
when you run sever.js icould help to run on the local machine
#### Any background context you want to provide?
this is the initial file to node js
 #### What are the relevant pivotal tracker stories?
link of pivot tracker: https://www.pivotaltracker.com/n/projects/2380400
story identification:#168404792

### CODE REVIEW

**Review by TheoOkafor — CHANGES_REQUESTED** [review]
Remove the `.env` from the repo.

File: server/app.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ import userRoutes from './routes/user.route';
 import swaggerUi from 'swagger-ui-express';
 import swaggerDoc from './swagger.json';
 
+import Database from './db/db';
+
+Database.createScripts();
```
Comment (TheoOkafor) [inline server/app.js:11]: I gave you feedback about this yesterday.
You shouldn't be doing this here

**Review by TheoOkafor — CHANGES_REQUESTED** [review]

File: server/app.js (hunk 1 above)
Comment (irakozeyves1) [reply server/app.js:11]: is one task but , there are configuration on packsagejson i install and some deletion of folder that , is look on staged area as many file but is one file of app only

Comment (irakozeyves1) [reply server/app.js:11]: am finish the to do it ,

Comment (irakozeyves1) [reply server/app.js:11]: can i merge it

Comment (irakozeyves1) [reply server/app.js:11]: could you check it again

**Comment by irakozeyves1** [comment]
not you see , am having the two databases for the test and for the
environment this is the , gate for environment but on my package json i
create the second for test that is created automatically

**Pull request merged**